packages/just-scripts/src/copy/CopyInstruction.ts

```diff
@@ -0,0 +1,177 @@
+import { join, basename, normalize } from 'path';
+import { readdirSync, statSync, existsSync } from 'fs';
+import { arrayify } from './arrayify';
+
+export interface CopyInstruction {
+  /**
+   * The path+filename of the source files. If more than one file is provided, the files will be merged in order
+   * and output to a file in the destination path.
+   */
+  sourceFilePath: string | string[];
+
+  /**
+   * The path+filename of the destination file.
+   */
+  destinationFilePath: string;
+}
+
+export interface CopyConfig {
+  copyInstructions: CopyInstruction[];
+}
+
+export interface LocalizedFileInfo {
```

Comment:
I think this might be too specific for now, let's remove the localized stuff

Reply:
kk